### ISSUE TITLE
Uncaught URIError: URI malformed

### ISSUE THREAD

**Issue by adamserafini** [mulesoft/api-console#528]
I have the following raml file which I build using `api-console build --json api.raml`

```
#%RAML 1.0
title: My API
version: v1
baseUri: http://somewebsite.com/
/restaurants/:
  get:
    queryParameters:
      city:
        type: string
        required: true
        example: berlin
    responses:
      200:
        body:
          application/json:
            example: |
              {
                "message" : "Hello World"
              }
```
I then copy the files the contents of /build folder to an ngnix server at http://mydocserver.com

The problem is at the point when I want to 'TRY' the api from the page hosted on the doc server. I get a message like in the following screenshot:

Any pointers where I'm going wrong?

![image](https://user-images.githubusercontent.com/1413003/30281698-69dae358-9713-11e7-889d-fa9e42cbaf25.png)

**Comment by adamserafini** [comment]
If it helps, the browser debug console reads as follows:

![image](https://user-images.githubusercontent.com/1413003/30281897-f3903486-9713-11e7-881a-7f0c4c62537a.png)

**Comment by jarrodek** [comment]
Hi,
Thank you for the issue report. 
It looks like Prism's library (highlights the response) throws an error for some reason (there must be malformed URL in the response). 
The problem is with the library but I'll try to do something about it.

**Comment by jarrodek** [comment]
@adamserafini Is it possible for you to send me an example response that throws this error?

**Comment by adamserafini** [comment]
Hi @jarrodek  I don't think there is a problem in the response because if I use a known 'correct' API the same things happens. For example check response here and screenshot below: https://jsonplaceholder.typicode.com/posts

Also what is weird - In the network tab it never makes a call to the URL https://jsonplaceholder.typicode.com/posts - is that expected? It is like something is going wrong before the requests even gets made.

![image](https://user-images.githubusercontent.com/1413003/30422228-9f1e7790-993f-11e7-9cd7-c540c93ed2f4.png)

**Comment by jarrodek** [comment]
This is now fixed in 6.0.0 as it has different approach to parsing response.
